plugins/item_tasks/web_client/stylesheets/paginateTasksWidget.styl

```diff
@@ -0,0 +1,12 @@
+.g-task-list-header
+  margin-bottom 10px
+
+  .g-task-pagination
+    ul
+      margin 0px
```

Comment:
This is more trivia than QA feedback, but when specifying a margin of 0, no unit is required 😄